### PR TITLE
Sf Radio

### PR DESCRIPTION
**Pull request by RakowskiPrzemyslaw**
# Related issue
https://github.com/DivanteLtd/storefront-ui/issues/27

# Scope of work
created radio component, api proposition
image

customized radio
image

# Screenshots of visual changes
image

customized radio
image

# Checklist

- [x] I followed [composition rules](https://github.com/DivanteLtd/storefront-ui/blob/master/docs/component-rules.md) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

**Comment by filrak** [comment]
What if ones want custom HTML for a certain option? For example like in designs:
image

**Comment by RakowskiPrzemyslaw** [comment]
@filrak thats why i provided "content" slot

**Comment by filrak** [comment]
Let me clarify. This is a common scenario so it should be docummented as a separate story i believe.

I guess the best option would be to just have content slot with multiple v-if's inside depending on option. 

Regarding API and implementation itself it looks very good ;)

File: src/components/atoms/SfRadio/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# SfRadio
+
+Component for simple group of radio buttons, pass an array get selected value via v-model
+
+## Props
```
Comment (filrak) [inline src/components/atoms/SfRadio/README.md]: shold be docummented in component as we discussed, don't duplicate ;)

File: src/components/atoms/SfRadio/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+- `value` - selected option
+- `options` - array of objects, keys (label) and (value) are required, more keys are optional
+
+## Events
+
+- `input` - event emmited when option is selected
+
+## Slots
```
Comment (filrak) [inline src/components/atoms/SfRadio/README.md]: same

File: src/components/atoms/SfRadio/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+## Events
+
+- `input` - event emmited when option is selected
+
+## Slots 
+
+- `content` - slot for your custom html option structure
+- `button` - slot for replacing default checkmark
+
+## CSS Modifiers
```
Comment (filrak) [inline src/components/atoms/SfRadio/README.md]: should be docummented as knobs (see other examples)

File: src/components/atoms/SfRadio/SfRadio.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+export default {
+  name: "SfRadio",
+  props: {
+    /**
+     * Array of objects, keys (label) and (value) are required, more keys are optional
```
Comment (filrak) [inline src/components/atoms/SfRadio/SfRadio.js]: "more keys are optional" -> what do you mean. It's confusing

File: src/components/atoms/SfRadio/SfRadio.scss [hunk 5]
```diff
@@ -0,0 +1,53 @@
+@import '../../../css/variables';
+
+$sf-radio__checkmark-size:  1.438rem !default;
```
Comment (filrak) [inline src/components/atoms/SfRadio/SfRadio.scss]: don;t use sf prefix for vars

File: src/components/atoms/SfRadio/README.md (hunk 3 above)
Comment (RakowskiPrzemyslaw) [reply src/components/atoms/SfRadio/README.md]: https://github.com/DivanteLtd/storefront-ui/pull/91/commits/0d33e8d6f260893ffb7cb1d968477c6ef880989d

File: src/components/atoms/SfRadio/README.md (hunk 2 above)
Comment (RakowskiPrzemyslaw) [reply src/components/atoms/SfRadio/README.md]: https://github.com/DivanteLtd/storefront-ui/pull/91/commits/0d33e8d6f260893ffb7cb1d968477c6ef880989d

File: src/components/atoms/SfRadio/README.md (hunk 1 above)
Comment (RakowskiPrzemyslaw) [reply src/components/atoms/SfRadio/README.md]: https://github.com/DivanteLtd/storefront-ui/pull/91/commits/0d33e8d6f260893ffb7cb1d968477c6ef880989d

File: src/components/atoms/SfRadio/README.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+Component for simple group of radio buttons, pass an array get selected value via v-model
+
+## Events
+
+- `input` - event emmited when option is selected
+
+<!-- Write down available CSS Modifiers -->
+
+## SCSS variables
```
Comment (filrak) [inline src/components/atoms/SfRadio/README.md]: just get rid of this one to not duplicate and I can marge ;)

Comment (RakowskiPrzemyslaw) [reply src/components/atoms/SfRadio/README.md]: https://github.com/DivanteLtd/storefront-ui/pull/91/commits/a56bc8e3c3d4610730d25da8c9f08c398541d31a

**Comment by filrak** [comment]
image
Not properly centered ;)

File: package.json [hunk 7]
```diff
@@ -34,7 +34,6 @@
     "@storybook/addon-links": "^5.0.11",
     "@storybook/addon-notes": "^5.0.11",
     "@storybook/addon-viewport": "^5.0.11",
-    "@storybook/addon-a11y": "^5.0.11",
```
Comment (filrak) [inline package.json]: why?

**Comment by filrak** [comment]
All stories look the same @RakowskiPrzemyslaw

File: package.json (hunk 7 above)
Comment (lorenaramonda) [reply package.json]: @filrak this was removed because it's duplicated I think 😉 there are two lines of the same module

**Pull request merged**